substratevm/src/com.oracle.svm.test/src/com/oracle/svm/test/jmx/JmxTest.java

```diff
@@ -89,7 +125,10 @@ private static MBeanServerConnection getLocalMBeanServerConnectionStatic() {
         try {
             JMXServiceURL jmxUrl = new JMXServiceURL("service:jmx:rmi:///jndi/rmi://" + "localhost" + ":" + TEST_PORT + "/jmxrmi");
             Map<String, Object> env = new HashMap<>();
-
+            String[] credentials = {"myrole", "MYP@SSWORD"}; // dummy password for testing
+            env.put(JMXConnector.CREDENTIALS, credentials);
+            // Include below if protecting registry with SSL
+            env.put("com.sun.jndi.rmi.factory.socket", new SslRMIClientSocketFactory());
```

Comment:
Should this key be a constant similar to the rest?

Reply:
Good point. I've changed it now.